pkg/fakerp/clients.go

```diff
@@ -0,0 +1,47 @@
+package fakerp
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	internalapi "github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/fakerp/client"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient"
+	"github.com/openshift/openshift-azure/pkg/util/azureclient/resources"
+)
+
+type clients struct {
```

Comment:
note: I could flatten this struct.. happy to do this if others think this looks better